pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -144,11 +144,7 @@ func haveOverlap(a1, a2 []string) bool {
 	if len(a1) > len(a2) {
 		a1, a2 = a2, a1
 	}
-	m := make(sets.String)
-
-	for _, val := range a1 {
-		m.Insert(val)
-	}
+	m := sets.New[string](a1...)
```

Comment:
```suggestion
	m := sets.New(a1...)
```

Does this work?

/hold

Reply:
yes ,done,thanks